niaid.bionimbus.org/etlMapping.yaml

```diff
@@ -1,11 +1,12 @@
 mappings:
-  - name: etl_niaid
+  - name: niaid
```

Comment:
isn't this a breaking change until we alias `niaid_4` as niaid?

Reply:
I don't think so, because the old alias still exists